client/client.go

```diff
@@ -1230,9 +1230,12 @@ func (client *OpenFgaClient) ReadChangesExecute(request SdkClientReadChangesRequ
 		req = req.ContinuationToken(*continuationToken)
 	}
 	requestBody := request.GetBody()
-	if requestBody != nil {
+	if requestBody.Type != "" {
 		req = req.Type_(requestBody.Type)
 	}
+	if !requestBody.StartTime.IsZero() {
+		req = req.StartTime(requestBody.StartTime)
+	}
```

Comment:
We'll need to guard against `requestBody` being nil here as it's valid to call `ReadChanges` with no body argument
```suggestion
	if requestBody != nil &&  requestBody.Type != "" {
		req = req.Type_(requestBody.Type)
	}
	if requestBody != nil &&  !requestBody.StartTime.IsZero() {
		req = req.StartTime(requestBody.StartTime)
	}
```